Add samples (#664)
dotnet/docs-maui: docs/get-started/resources.md

  Learn how to build a .NET MAUI app that displays a list of monkeys from around the world. You'll start by building the business logic backend that retrieves JSON-encoded data from a REST-based endpoint. You then use .NET MAUI to find the closest monkey to you, and show the monkey on a map. You'll also examine how to display data using different approaches, and then finally fully theme the app.

> - [.NET MAUI Samples](https://docs.microsoft.com/samples/browse/?expanded=dotnet&products=dotnet-maui)
>   
>   Explore .NET MAUI samples and full example application and try them out locally.
>   

Reviewer: Fix link and rewrite the description a bit.

```suggestion
- [.NET MAUI Samples](/samples/browse/?expanded=dotnet&products=dotnet-maui)
  
  Download and explore the code of different example .NET MAUI applications.
  
```
